sympy/__init__.py

```diff
@@ -62,6 +62,10 @@ def __sympy_debug():
         evalf, Tuple, Dict, gcd_terms, factor_terms, factor_nc, evaluate,
         Catalan, EulerGamma, GoldenRatio, TribonacciConstant)
 
+from .core.add import add
+from .core.mul import mul
+from .core.power import power
```

Comment:
If this is experimental then perhaps it should not be included in the main `__init__.py` (and either way, this should be added to `sympy/core/__init__.py` the existing `from .core` line here).